tests/conftest.py

```diff
@@ -1,10 +1,8 @@
 from typing import Dict, Tuple, Type, Union, cast
 
 import pytest
-from django.test.client import (
-    AsyncClient,  # type: ignore
-    Client,
-)
+from django.test.client import AsyncClient  # type: ignore
+from django.test.client import Client
```

Comment:
lint is still failing, I think is because of this change here

Reply:
@bellini666 fixed again. I think black was fighting ruff :/